config_models/models.py

```diff
@@ -9,7 +9,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from rest_framework.utils import model_meta
-
+import six
```

Comment:
I don't see `six` among the dependencies in `setup.py` yet.  Maybe just use the version bundled with `django.utils` as you used in the file below?

Reply:
Yeah; fixed now on last commit (imports from django.utils)